### ISSUE TITLE
Testing components containing Sveltestrap fails tests (using svelte-testing-library)

### ISSUE THREAD

**Issue by x-GF-x** [bestguy/sveltestrap#437]
Hello,
I have a component I am testing using svelte-testing-library. I am receiving this error:  
```
TypeError: Cannot read property '$$' of undefined
      at tooltipEl (node_modules/sveltestrap/node_modules/svelte/internal/index.mjs:1863:32)
      at new Object.<anonymous>.t.FormGroup (node_modules/sveltestrap/src/Tooltip.svelte:126:18)
```
When I comment out the Sveltestrap components (FormGroup, InputGroup, Input) the test passes fine.
Any help would be appreciated.
Thank you.

**Comment by bestguy** [comment]
Hi, we are using `@testing-library/svelte` in this library without issues:

https://github.com/bestguy/sveltestrap/blob/master/src/__test__/Alert.spec.js#L2

Can you send a repo or sample of the test that is failing?

**Comment by x-GF-x** [comment]
Of course, here you go: https://svelte.dev/repl/c36932992d7f4adcb9329ed6b1deff5e?version=3.46.4

**Comment by bestguy** [comment]
Hi, I think you need to make this repo public, seeing a 404

**Comment by bestguy** [comment]
Hi @x-GF-x ,
Not sure the cause yet, but this seems to have started with sveltestrap@5.8.2.  
As a temporary workaround you can revert to 5.8.1 as there should be no functional changes:
`npm install sveltestrap@5.8.1`

I'll do some further comparisons and try and get a new release out asap if I can find the issue, thanks for the heads up

**Comment by x-GF-x** [comment]
Hey, no problem at all. Thanks for the help!

**Comment by x-GF-x** [comment]
@bestguy The fix works so I'll go ahead and close. Thanks for the prompt response and the great library.
